package.json

```diff
@@ -5,22 +5,23 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "watch": "concurrently -c gray,blue -k -n hugo,tailwind \"npm run watch --workspace hugo\" \"npm run watch --workspace tailwind\"",
-    "watch:gitpod": "concurrently -c gray,blue -k -n hugo,tailwind \"npm run watch:gitpod --workspace hugo\" \"npm run watch --workspace tailwind\"",
-    "build": "npm run clean && npm run build --workspace=tailwind --workspace=hugo",
-    "clean": "shx rm -rf public && npm run clean --workspace=tailwind"
+    "watch": "npm run build --workspace foundation && concurrently -c gray,blue -k -n hugo,tailwind \"npm run watch --workspace hugo\" \"npm run watch --workspace tailwind\"",
+    "watch:gitpod": "npm run build --workspace foundation && concurrently -c gray,blue -k -n hugo,tailwind \"npm run watch:gitpod --workspace hugo\" \"npm run watch --workspace tailwind\"",
+    "build": "npm run clean && npm run build --workspace foundation --workspace=tailwind --workspace=hugo",
+    "clean": "shx rm -rf public && npm run clean --workspace foundation --workspace=tailwind"
   },
   "devDependencies": {
-    "concurrently": "~7.6.0",
+    "concurrently": "~8.2.1",
```

Comment:
Noting that concurrently v8.X.X dropped node12 support in favor of node18, which works just fine for us here.

Reply:
Thanks.